src/app/helptext/system/reporting.ts

```diff
@@ -30,7 +30,13 @@ reset_button: T('Reset to Defaults'),
 dialog: {
     title: T('Change Settings and Clear Report History?'),
     message: T('Report history is cleared when <i>CPU reporting</i>, <i>Graph Age</i>, \
- or <i>Graph Points</i> are changed.'),
+<i>Graphite Separate Instances</i> or <i>Graph Points</i> are changed.'),
     action: T('Continue')
-}
+},
+graphite_separateinstances_placeholder: T('Graphite Separate Instances'),
+graphite_separateinstances_tooltip:T("If checked, when sending to Graphite,\
```

Comment:
```suggestion
graphite_separateinstances_tooltip:T("Enabling sends the \
 <i>plugin instance</i> and <i>type instance</i> to Graphite as separate \
 path components: <i>host.cpu.0.cpu.idle</i>.<br><br> Disabling sends \
 the <i>plugin</i> and <i>plugin instance</i> as one path component and \
 <i>type</i> and <i>type instance</i> as another component: \
 <i>host.cpu-0.cpu-idle</i>.")
```
Suggestion unifies tense, removes asides and pauses, and adds an empty space between set/unset descriptions to improve readability.